sqlmesh/core/renderer.py

```diff
@@ -114,6 +113,16 @@ def _render(
                 table_mapping=table_mapping,
                 deployability_index=deployability_index,
                 default_catalog=self._default_catalog,
+                table=lambda e, comments=True: self._resolve_tables(
```

Comment:
A few comments:
1. Calling `resolve_tables` seems like an overkill. Provided we only need this line:
```
mapping = {**to_table_mapping(snapshots.values(), deployability_index), **table_mapping}
```
and normalization.

2. I don't like the name `table`. Can we be more specific? Eg. `physical_table` or `resolve_table`?

Reply:
3. Seems like our built-in macros are all over the place. Can this be a macro in `macros.py`?

Reply:
Btw, why do we need this 2nd argument in the lambda?